### PR TITLE
Pretty Face App ready!

### PR DESCRIPTION
**Pull request by ignacioinnovo**
*Thanks for submitting a PR to the Snppts ✂️ library!*

## Submission description

I have donde some improvements since the mail I sended to Gerald. Hope you like them!

### Possible platform improvements

The idea and implementation of snppts.dev is great. Congratulations!

### CODE REVIEW

**Comment by jfversluis** [comment]
It seems I can’t update the branch with the latest stuff from master. Would you be able to do that?

**Comment by ignacioinnovo** [comment]
I clicked on Update branch from master and I now have an error: "Merging is blocked: Merging can be performed automatically wuth 1 approving review". Please, let me know if I can help.

File: src/Snppts/Snippets/PrettyFaceApp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        new Uri("https://raw.githubusercontent.com/InnovoTecnologias/PrettyFaceApp/master/Media/Preview1.jpg"),
+        new Uri("https://raw.githubusercontent.com/InnovoTecnologias/PrettyFaceApp/master/Media/Preview2.jpg"),
+        new Uri("https://raw.githubusercontent.com/InnovoTecnologias/PrettyFaceApp/master/Media/Preview3.jpg"),
+        new Uri("https://raw.githubusercontent.com/InnovoTecnologias/PrettyFaceApp/master/Media/Preview4.jpg"),
+        new Uri("https://raw.githubusercontent.com/InnovoTecnologias/PrettyFaceApp/master/Media/Preview5.jpg")
+    };
+
+    public IList<Category> Categories => new List<Category>
+    {
+        Category.NAVIGATIONS, Category.DASHBOARD, Category.LAUNCHSCREEN, Category.CARDS, Category.LISTS, Category.DASHBOARD
```
Comment (jfversluis) [inline src/Snppts/Snippets/PrettyFaceApp]: You have dashboard 2 times

File: src/Snppts/Snippets/PrettyFaceApp [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    public string GithubRepoName => "InnovoTecnologias/PrettyFaceApp";
+    public bool ContainsAndroidSample => true;
+    public bool ContainsiOSSample => true;
+    public bool ContainsUWPSample => false;
+
+    public string Description => "This snippet was created to give you a complete starting point for your app.";
+
+    public IAmAnAuthor AuthorInfo => new InnovoTecnologias();
+    
+    public Uri ExternalUri => new Uri("https://github.com/InnovoTecnologias/PrettyFaceApp");
```
Comment (jfversluis) [inline src/Snppts/Snippets/PrettyFaceApp]: Please use the `GitHubRepoInfo` property

**Review by jfversluis — CHANGES_REQUESTED** [review]

File: src/Snppts/Snippets/PrettyFaceApp (hunk 1 above)
Comment (ignacioinnovo) [reply src/Snppts/Snippets/PrettyFaceApp]: Sorry for this error.

File: src/Snppts/Snippets/PrettyFaceApp (hunk 2 above)
Comment (ignacioinnovo) [reply src/Snppts/Snippets/PrettyFaceApp]: Done.

**Review by jfversluis — APPROVED** [review]

**Pull request merged**